ocp_resources/resource.py

```diff
@@ -391,9 +391,9 @@ def __init__(
         self.dry_run = dry_run
         self.node_selector = node_selector
         self.node_selector_labels = node_selector_labels
-        self.node_selector_spec = self._node_selector_spec_prepare()
+        self.node_selector_spec = self._prepare_selector_spec()
 
-    def _node_selector_spec_prepare(self):
+    def _prepare_selector_spec(self):
```

Comment:
I hope the last one :)
Why did you remove the work node from the name?

Reply:
I don't have any nice words to describe this mistake